app/helpers/categories_helper.rb

```diff
@@ -6,7 +6,7 @@ def show_table(root, first, last)
           '</li>'+"\n"
     end
 
-    categories = (root.children.to_a).sort_by!{ |cat| cat.list_order }
+    categories = (root.children.to_a).sort_by!{ |cat| cat.list_order || cat.name }  ## sort by alpha if list_order is nil -dji
```

Comment:
When I test this with a mixed array with some items having a nil list_order, I get the following error:
```
ArgumentError: comparison of Fixnum with String failed
```